projects/addon-editor/components/toolbar-tools/full-color-picker/highlight-color/highlight-full-color-picker.module.ts

```diff
@@ -0,0 +1,34 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {
+    TuiColorPickerModule,
+    TuiColorSelectorModule,
+    TuiPaletteModule,
+} from '@taiga-ui/addon-editor/components/color-selector';
+import {TuiActiveZoneModule, TuiLetModule} from '@taiga-ui/cdk';
+import {
+    TuiButtonModule,
+    TuiDropdownModule,
+    TuiHintModule,
+    TuiHostedDropdownModule,
+} from '@taiga-ui/core';
+
+import {TuiHighlightFullColorPickerComponent} from './highlight-full-color-picker.component';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        TuiHostedDropdownModule,
+        TuiButtonModule,
+        TuiPaletteModule,
+        TuiHintModule,
+        TuiDropdownModule,
+        TuiActiveZoneModule,
+        TuiLetModule,
+        TuiColorPickerModule,
```

Comment:
Are `TuiColorPickerModule` and `TuiPaletteModule` actually required here ?

Reply:
fixed